python/lsst/afw/coord/refraction.py

```diff
@@ -212,8 +212,8 @@ def humidityToPressure(weather):
 
     Returns
     -------
-    `float`
-        The water vapor pressure in millibar
+    `astropy.units.quantity.Quantity`
```

Comment:
I suggest `astropy.units.Quantity` since that is the documented version. The existence of a `quantity` sub-module appears to be an internal detail of astropy.